content/post/34-go-ipfs-0.4.14.md

```diff
@@ -0,0 +1,69 @@
+---
+date: 2018-03-21
+url: 34-go-ipfs-0.4.14
+title: go-ipfs 0.4.14 released
+author: Victor Bjelkholm
+---
+
+[go-ipfs 0.4.14](https://dist.ipfs.io/#go-ipfs) has been released today. Not
+only have we improved memory and CPU usage but we also managed to fix a lot of bugs,
+ship a major improvement to IPNS performance and lots of refactoring! \o/
```

Comment:
though maybe tone the ipns bit down. Its still experimental, don't want people thinking it applies everywhere yet.

Reply:
"ship a major experimental improvement to IPNS performance" sounds better? I know a lot of people are waiting to hear this, but we don't want to over-hype it.